### PR TITLE
Fix #3552 option_story_text_size.xml 

### LINKED ISSUE #3552: Merge option_story_text_size.xml into single xml file
Currently there are 3 versions of `option_story_text_size.xml` file merge it into single xml file.

We can use https://text-compare.com/ to compare two versions of this file and for all the differences we can create variables in `dimens.xml` file and use it accordingly.

**Note**: In PR, make sure you add before and after screenshot of mobile-portrait, mobile-landscape, tablet-portrait and tablet-landscape for comparison and make sure that there is not difference between before and after UI.

### PR DESCRIPTION
**Pull request by shankarpriyank**
## Explanation
- Merged 2 files of  option_story_text_size.xml 
 - Fixes #3552

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

land
potrait
rtl land
rtl

### CODE REVIEW

**Comment by bkaur-bkj** [comment]
@shankarpriyank Write PTAL @reviewe name  in the comments not in the explanation, do edit it

**Comment by shankarpriyank** [comment]
PTAL @bkaur-bkj

**Review by bkaur-bkj — APPROVED** [review]
The changes LGTM. @rt4914 PTAL

**Review by rt4914 — COMMENTED** [review]
Thanks @shankarpriyank 

I think sw600dp/option_story_text_size can also be merged.

Basically use the code of `sw600dp/option_story_text_size` in `layout/option_story_text_size.xml` and also in `sw600dp/dimens.xml` file use following:

```
<!-- StoryTextSize -->
  <dimen name="option_story_text_size_padding_start">16dp</dimen>
  <dimen name="option_story_text_size_padding_end">16dp</dimen>
```

File: app/src/main/res/values/dimens.xml [hunk 1]
```diff
@@ -605,4 +605,8 @@
   <dimen name="topic_lessons_title_margin_start">32dp</dimen>
   <dimen name="topic_lessons_title_margin_top">32dp</dimen>
   <dimen name="topic_lessons_title_margin_end">32dp</dimen>
+
+  <!-- StoryTextSize -->
```
Comment (rt4914) [inline app/src/main/res/values/dimens.xml]: Update to `Options story text size` as this is not a file instead just a regular comment. For files we use `AbcXyz`

**Review by rt4914 — CHANGES_REQUESTED** [review]
PTAL.  Also please recreate the after screenshots once these changes are done.

**Comment by bkaur-bkj** [comment]
> Thanks @shankarpriyank
> 
> I think sw600dp/option_story_text_size can also be merged.
> 
> Basically use the code of `sw600dp/option_story_text_size` in `layout/option_story_text_size.xml` and also in `sw600dp/dimens.xml` file use following:
> 
> ```
> 
>   <dimen name="option_story_text_size_padding_start">16dp</dimen>
>   <dimen name="option_story_text_size_padding_end">16dp</dimen>
> ```

@rt4914 I wanted to know that how is it possible to access a property from viewmodel if it has been not initialised for mobile xmls.

**Comment by shankarpriyank** [comment]
@rt4914 There are two files for the tablet layout(landscape,potrait), should I add dimens in both of them?

**Comment by rt4914** [comment]
> > Thanks @shankarpriyank
> > I think sw600dp/option_story_text_size can also be merged.
> > Basically use the code of `sw600dp/option_story_text_size` in `layout/option_story_text_size.xml` and also in `sw600dp/dimens.xml` file use following:
> > ```
> > 
> >   <dimen name="option_story_text_size_padding_start">16dp</dimen>
> >   <dimen name="option_story_text_size_padding_end">16dp</dimen>
> > ```
> 
> @rt4914 I wanted to know that how is it possible to access a property from viewmodel if it has been not initialised for mobile xmls.

@bkaur-bkj If the values are not initialised then we cannot use it. But if we thing that merging files can become easy, in that case you can inititalse the value in ViewModel so that you can use it in xml files.

**Comment by rt4914** [comment]
> @rt4914 There are two files for the tablet layout(landscape,potrait), should I add dimens in both of them?

@shankarpriyank If needed then yes add it in both files.

**Comment by bkaur-bkj** [comment]
> > > Thanks @shankarpriyank
> > > I think sw600dp/option_story_text_size can also be merged.
> > > Basically use the code of `sw600dp/option_story_text_size` in `layout/option_story_text_size.xml` and also in `sw600dp/dimens.xml` file use following:
> > > ```
> > > 
> > >   <dimen name="option_story_text_size_padding_start">16dp</dimen>
> > >   <dimen name="option_story_text_size_padding_end">16dp</dimen>
> > > ```
> > 
> > 
> > @rt4914 I wanted to know that how is it possible to access a property from viewmodel if it has been not initialised for mobile xmls.
> 
> @bkaur-bkj If the values are not initialised then we cannot use it. But if we thing that merging files can become easy, in that case you can inititalse the value in ViewModel so that you can use it in xml files.

Okay got it

File: app/src/main/res/values/dimens.xml [hunk 2]
```diff
@@ -625,4 +625,8 @@
   <dimen name="topic_lessons_title_margin_start">32dp</dimen>
   <dimen name="topic_lessons_title_margin_top">32dp</dimen>
   <dimen name="topic_lessons_title_margin_end">32dp</dimen>
+
+  <!-- Options story text size  -->
```
Comment (rt4914) [inline app/src/main/res/values/dimens.xml:629]: Update to `<!-- Options story text size -->` . There is one extra space just after `size`. Ditto change for other xml files.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@shankarpriyank There are couple of issues with this PR. 

1. Looks like your branch has been deleted as I am unable go to your branch.
2. There are changes which should not be part of this PR. See "File Changed" section in github. To fix this either merge with latest develop from oppia or undo the changes.

**Comment by rt4914** [comment]
@bkaur-bkj How did this PR work for you? I am unable to check the branch. Looks like it has been deleted and also all theCI checks are stopped is weird.

**Comment by shankarpriyank** [comment]
> @shankarpriyank There are couple of issues with this PR.
> 
> 1. Looks like your branch has been deleted as I am unable go to your branch.
> 2. There are changes which should not be part of this PR. See "File Changed" section in github. To fix this either merge with latest develop from oppia or undo the changes.

Yes I actually deleted this repo, I had a talk with my mentors ,  I messed up the branch real bad , so and it was suggested  that it would  be best if I make a new pr for this issue

**Comment by shankarpriyank** [comment]
Also I actually merged all the three files in one , but It was not working as intended , I will attach the screenshots in a fresh pr

**Pull request closed without merging**